Networking/Networking/Model/ShippingLabel/ShippingLabelAddressValidationError.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+/// Represents Shipping Label Address Validation Error.
+///
+public struct ShippingLabelAddressValidationError: Equatable {
+    public let address: String?
+    public let general: String?
```

Comment:
Good point. The name is definitely not very descriptive. I think renaming it is a good idea

Reply:
Yes, I tried to follow the model as much as possible because the struct already contains the word error. But for sure I can rename them to `addressError` and `generalError`. 👍